torch_geometric/nn/conv/simple_conv.py

```diff
@@ -0,0 +1,91 @@
+from typing import List, Optional, Union
+
+import torch
+from torch import Tensor
+
+from torch_geometric.nn.aggr import Aggregation
+from torch_geometric.nn.conv import MessagePassing
+from torch_geometric.typing import (
+    Adj,
+    OptPairTensor,
+    OptTensor,
+    Size,
+    SparseTensor,
+)
+from torch_geometric.utils import spmm
+
+
+class SimpleConv(MessagePassing):
+    r"""A simple message passing operator that performs (non-trainable)
+    propagation.
+
+    .. math::
+        \mathbf{x}^{\prime}_i = \bigopolus_{j \in \mathcal{N(i)}} e_{ji} \cdot
+        \mathbf{x}_j
+
+    Args:
+        aggr (str or [str] or Aggregation, optional): The aggregation scheme
+            to use, *e.g.*, :obj:`"add"`, :obj:`"sum"` :obj:`"mean"`,
+            :obj:`"min"`, :obj:`"max"` or :obj:`"mul"`.
+            In addition, can be any
+            :class:`~torch_geometric.nn.aggr.Aggregation` module (or any string
+            that automatically resolves to it). (default: :obj:`"sum"`)
+        combine_root (str, optional): Specified how to combine the central node
```

Comment:
I'm a little uncomfortable with name `combine_root`. I think we can simply call it `kernel`. The kernel is used to combine representations. The kernel could be a `sum` kernel or `concat` kernel.

Reply:
I think `kernel` is confusing, especially since we already "combine" representations of neighbors. Any other idea? :)

Reply:
how about `concat (bool, default False)`? or `fuse= sum / concat`? Otherwise we go with `combine_root`.

Reply:
Mh, `concat` wouldn't work since I think we want to support both sum/concat. `fuse` would be irritating since I think of operator fusion in this case :D What about `group`/`group_root`/`combine`/`combine_root`?